### DISCUSSION TITLE
is it possible to do cryptosmite using e-halcon.

### DISCUSSION THREAD

**Discussion by B06e** [3kh0/ext-remover#1062] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I was wondering if I could set up cryptosmite using e-halcon because of the downgrade.

**Comment by AshtonDavies** [comment, marked as answer]
Involving E-Halcyon would require you to jumpstart after every reboot.

**Comment by B06e** [comment]
I was thinking using the older version of chromeos so that cryptosmite could work. Since it's a older version of chromeos when you use e-halcon.

**Reply by AshtonDavies** [reply]
The kernel would remain at the same version, so that would not work.
